### PR TITLE
Address bar update

### PR DESCRIPTION
**Pull request by mrtmrt233**
Implements update proposed [here](https://github.com/Public-Tree-Map/public-tree-map/issues/278#issuecomment-632276298)

Address bar now drops a pin at target location.

Map panning from one location to another is a bit off, might look into later. Current state seem publishable? @emillipede

### CODE REVIEW

**Comment by emillipede** [comment]
thank you @mrtmrt233 ! this looks great.

two questions:
- is it possible to maintain something like the feature that shows a user's direction and/or adjusts the user's location marker as they walk down a street?  
File
- what do you mean about panning? I noticed that when I type an address into the search bar and click to move to it, the intended location winds up at the very bottom right corner of the page (off the visible screen)

thanks again!

**Comment by mrtmrt233** [comment]
Yes, should be doable and yes that's what I'm referring to.

**Comment by emillipede** [comment]
Thank you! Maintaining a direction indicator and movement updates would be excellent.

**Comment by mrtmrt233** [comment]
now uses leaflet.control.locate for current location.

**Comment by emillipede** [comment]
thank you @mrtmrt233 it's already a huge improvement.

a few more (new) questions:
- can the marker track motion? It seems like it is built to do that now, but I tried to move around and couldn't get the marker to follow me or move at all
- if motion tracking/following is an option, can the marker include a motion/direction indicator? something like the arrow in the screenshot above.
- is it possible for a user to re-select the location sensor? eg if they zoom to their location, then pan away, and then want to zoom back to their location? I tried to re-click the icon, and it didn't respond. if I switch away from the tab, though, I'll get a new request to see my location

please let me know what you think. thanks again

**Comment by mrtmrt233** [comment]
Location live tracking is on by default, but the arrow icon wasn't displayed. Fixed in latest push, along with refactoring and readability improvements.

Also fixed commit history

File: public/js/Map.js [hunk 1]
```diff
@@ -62,7 +58,8 @@ var app = this.app || {};
       keepCurrentZoomLevel: true,
```
Comment (emillipede) [inline public/js/Map.js:58]: please update to false and zoom in to the user's location. I think the zoom level on click for tree markers is 14.2.

**Comment by emillipede** [comment]
thank you @mrtmrt233! 

can you please share a screenshot? when I test the branch on a web browser, I can't get the arrow to show up

**Comment by mrtmrt233** [comment]
Should be working...
IMG_592C984FAA00-1

What's the testing setup when the arrow failed to show up? Did the browser prompt 2 requests for both location and motion & orientation?

**Comment by emillipede** [comment]
thank you @mrtmrt233 

I didn't get the second prompt about motion and orientation. I'm test the branch by opening the index.html file in chrome or firefox. the screenshot look great. I really appreciate your patience with my questions. I think this PR makes a big improvement to the site. thank you again.

File: public/js/Map.js [hunk 2]
```diff
@@ -48,12 +70,8 @@ var app = this.app || {};
...
       },
-      onLocationOutsideMapBounds: control => {
-          control.stop();
-          alert(control.options.strings.outsideMapBoundsMsg);
-      },
       strings: {
-        outsideMapBoundsMsg: `Sorry, we only document tree data in Santa Monica.`,
+        popup:'Current Location',
         needsPermissionMsg: `You need to grant your browser access 
         to your location in order to use this feature.`
```
Comment (emillipede) [inline public/js/Map.js:76]: sorry! I was about to merge and realized I want to ask for (or can make, if you prefer) one more small change. Can you please update line 75 to read "In order to use this feature, please grant your browser location access. Public Tree Map will not store your location information." I don't think the line break at 75/76 is necessary, and in my experience it makes the permission message hard to read

**Comment by mrtmrt233** [comment]
If the app is not used on mobile there's no motion & orientation functionality available I think

**Comment by emillipede** [comment]
thank you @mrtmrt233 !!

**Pull request merged**